index.html

```diff
@@ -3197,7 +3110,274 @@ <h2>
 document</a>.
       </p>
     </section>
-  </section>
+</section>
+    <section>
+      <h2>
+JSON LD Description and Extension
+      </h2>
+    
+      <p class="note" title="JSON-LD Description">
+The following text was previously in the section on serialized representations. 
+It has been moved to this appendix to allow the text to be re-incorporated into
+the document in more appropriate sections or removed entirely, as required by
+the working group.
+      </p>
+      
+            <p>
+      [[!JSON-LD]] is a JSON-based format used to serialize
```

Comment:
Indentation for this entire section is off, please fix. I know we're just moving it, but sometimes that doesn't end up happening and it gets splattered around the specification. We need to keep the formatting consistent to make the lives of people editing the spec and putting in PRs easier.

Reply:
OK, I'll fix this -- it was copied from the previous version as-is.